src/dvclive/live.py

```diff
@@ -7,8 +7,19 @@
 import os
 import shutil
 import tempfile
+
 from pathlib import Path, PurePath
-from typing import Any, Dict, List, Optional, Set, Tuple, Union, TYPE_CHECKING, Literal
+from typing import (
```

Comment:
does it change only formatting ? not critical, but can we do it please in a separate PR? (not a blocker, minor)